shopinvader/models/product_product.py

```diff
@@ -35,12 +35,25 @@ def _search_shopinvader_backend_ids(self, operator, value):
         compute="_compute_is_shopinvader_binded",
         store=True,
         index=True,
-        help="Technical field to know if this product is related by a"
+        help="Technical field to know if this product is related by a "
         "(at least one) shopinvader backend",
     )
 
+    is_shopinvader_active_binded = fields.Boolean(
```

Comment:
```suggestion
    is_shopinvader_active_bound = fields.Boolean(
```
`s/binded/bound` all around pls :pray: :wink:

Reply:
feel free to do it only for fwd ports :stuck_out_tongue_winking_eye:

Reply:
Sorry for the delay,

I agree but  that would mean renaming the existing `is_shopinvader_binded` and make a migration script for it I suppose.

Also it would mean renaming each reference to it everywhere in the code.
I'd prefer doing it in a separate PR tbh